packages/board-server/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:20
+COPY --from=breadboard . /breadboard/
+# For the time being, we will assume container deployments are exclusively sqlite.
+ENV STORAGE_BACKEND "sqlite"
+ENV GOOGLE_CLOUD_PROJECT "n/a"
```

Comment:
I think we need to set ALLOWED_ORIGINS here for the board server to accept requests. (Similar to what we do in `app.yaml`. Since each deployment will need a different value, I've just set it to empty string so that the person deploying has a placeholder to fill in.

By default it will reject all connections.

Reply:
Ah, excellent spot. Ok, I will investigate.